src/components/Dao/ProposalSchemeCard.tsx

```diff
@@ -91,17 +91,14 @@ export default withSubscription({
     const arc = getArc();
     const dao = arc.dao(props.dao.address);
     return combineLatest(
-      props.scheme.state(),
+      props.scheme.state(standardPolling()),
```

Comment:
why adding standardPolling here ?

Reply:
because this scheme state would not update otherwise, and we need that or else the scheme proposal count properties will not be responsive to changes as proposals move through their lifecycle.